Header header
MatrixProto ProtoMsg
int32 ImageCount
MatrixImage[] ImageMsg
int32 ParsingCount
MatrixParsing[] ParsingMsg